From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Mon, 16 Jan 2023 23:22:18 +0100
Subject: Don't look for build.properties in the user home directory.

Forwarded: not-needed

This directory doesn't exist on the builders and the attempt to load
a property file there causes a build failure.
---
 build.xml | 1 -
 1 file changed, 1 deletion(-)

--- a/build.xml
+++ b/build.xml
@@ -847,7 +847,6 @@
     <!-- Property that determines if manifests need updating -->
     <uptodate property="manifests.uptodate"
         targetfile="${tomcat.manifests}/default.manifest" >
-      <srcfiles file="${user.home}/build.properties" />
       <srcfiles file="${basedir}/build.properties" />
       <srcfiles file="${basedir}/build.properties.default" />
       <srcfiles file="${basedir}/build.xml" />
